CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Pin arduino-esp32 package to version 1.0.5 to fix build issue
```

Comment:
Ahh. You are using 1.0.6, but this changelog entry is wrong.

Reply:
Apologies for typo, should be fixed up now

Reply:
Thanks for the PR guys. 

I'm away from my main computer (I'm checking in from a quick break from class), can anyone confirm what version we have been building with? If anyone can confirm it's 1.06, I'm good with the merge. No really need to keep develop branch absolutely rock stable as I will build auto update binaries from master.

Reply:
@doudar We had been building with master, so who knows...